### PR TITLE
Log useful account commands when user auths a new account

### PR DESCRIPTION
**Pull request by brandenrodgers**
## Description and Context

This is an idea to help make account switching in the CLI easier to understand and more discoverable:
- After the user runs `hs auth` or `hs init` we log out some additional commands that help make account handling easier. 
- I updated the prompts + copy in the flow to (hopefully) make them easier to understand. 
- The user now has the option to automatically set the newly connected account as the default when running `ha auth`

### hs init (OLD)
image

### hs init (NEW)
image

### hs auth (OLD)
image

### hs auth (NEW) w/ setting as default
image

### hs auth (NEW) w/out setting as default
image

### CODE REVIEW

**Comment by brandenrodgers** [comment]
### Update

I updated the spacing of this a little and added this same "feature highlight" to the `hs init` command too:

#### hs init
Screen Shot 2022-05-04 at 6 11 51 PM

#### hs auth
Screen Shot 2022-05-04 at 5 13 02 PM

**Comment by brandenrodgers** [comment]
Latest iteration:
image (2)

File: packages/cli/commands/sandbox/create.js [hunk 1]
```diff
@@ -85,7 +53,16 @@ const personalAccessKeyFlow = async (env, account, name) => {
     process.exit(EXIT_CODES.SUCCESS);
   }
 
-  const validName = await promptForAccountNameIfNotSet(updatedConfig, name);
+  let validName = updatedConfig.name;
```
Comment (brandenrodgers) [inline packages/cli/commands/sandbox/create.js:56]: @adamawang just giving you the heads up. I tried to DRY things up a little bit by sharing prompts between this command and the auth command. I also added the new `hs auth` logs to the "personalAccessKeyFlow" portion of this command. If you get the chance, I'd love some eyes on the changes in this file to make sure everything makes sense.

Comment (adamawang) [reply packages/cli/commands/sandbox/create.js:56]: Everything looks perfect, thanks for the heads up! 🎉

**Comment by TheWebTech** [comment]
This is great. Too many folks don't know about `hs accounts use`. This should help with adoption.

**Pull request merged**